doc/nrf/known_issues.rst

```diff
@@ -210,6 +210,14 @@ Subsystems
 |BLE|
 =====
 
+.. rst-class:: v1-4-0 v1-3-2 v1-3-1 v1-3-0
+
+NCSDK-6844: Security procedure failure can terminate GATT client request
+  A security procedure terminates the current GATT client request that is currently in progress, unless the current request was the reason to initiate the security procedure.
```

Comment:
That are a lot of currents in there now ... ;)
Either
```suggestion
  A security procedure terminates the current GATT client request, unless the current request was the reason to initiate the security procedure.
```
or 
```suggestion
  A security procedure terminates the GATT client request that is currently in progress, unless the current request was the reason to initiate the security procedure.
```

Reply:
Removed 2x "current", the second one seemed redundant to me as well.